README.md

```diff
@@ -270,6 +270,47 @@ awxJob:
     }
 ```
 
+### Alertmanager Configuration
+
+Follow the upstream [Prometheus Alertmanager documentation](https://prometheus.io/docs/alerting/configuration/)
+to configure alerts.
+
+For reference, here is an example Alertmanager configuration that sends
+an alert to Autoheal with authentication. This example assumes autoheal
+and the Alertmanager are running on the same OpenShift cluster,
+and requires Alertmanager 0.15 or newer.
+
+```yaml
+global:
+  resolve_timeout: 1m
+
+route:
+  group_wait: 1s
+  group_interval: 1s
+  repeat_interval: 5m
+  receiver: autoheal
+  routes:
+  - match:
+      alertname: DeadMansSwitch
+    repeat_interval: 5m
+    receiver: autoheal 
+receivers:
+- name: default
+- name: deadmansswitch
+- name: autoheal
+  webhook_configs:
+  - url: https://autoheal.openshift-autoheal.svc/alerts
+    http_config:
+      bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+      ca_file: /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+```
+
+When using the cluster-monitoring-operator, save the configuration as `alertmanager.yaml` and use this command to apply it:
+
+`oc -n openshift-monitoring create secret generic alertmanager-main --from-literal=alertmanager.yaml="$(< alertmanager.yaml)" --dry-run -oyaml | oc -n openshift-monitoring replace secret --filename=-`
```

Comment:
This is more readable if you use a code block, and split the lines:

````
```
oc create secret generic alertmanager-main \
--namespace openshift-monitoring \
--from-literal=... \
--dry-run \
--output=yaml \
| \
oc replace secret \
--namespace=openshift-monitoring \
--filename=-
```
````

Reply:
I copied this from the CoreOS monitoring documentation (and just modified it for OpenShift), I think it's better as a one-liner because it's easier to copy and paste - but I can change this.

Reply:
Done.